<launch>

  <node pkg="love_maker_2000" type="brain.py" name="movement_controller" output="screen">
  </node>

</launch>
